README.md

```diff
@@ -316,27 +251,31 @@ Backwards incompatible changes to *Implicit* APIs that have been included in a R
 
-*Primary* Platforms consist of those supported in LTS Releases. These are the Platforms on which Node.js must not break and for which subsequent LTS Releases must be fully operational with no regressions.
+*Supported* platforms consist of those with support in the `master` branch. For every release, the build must complete successfully and pass a full CI test run on every *Supported* platform.
 
-*Secondary* Platforms consist of those with support in the master branch but have not yet been included in an LTS Release.
+*Experimental* Platforms consist of those for which there is active ongoing development to port Node.js to the platform but functionality may not yet be feature complete or fully operational. Support for experimental platforms may not yet have been committed to `master` and might be done in a Development branch or separate repository entirely.
 
-*Experimental* Platforms consist of those for which there is active ongoing development to port Node.js to the platform but functionality may not yet be feature complete or fully operational. Support for experimental platforms may not yet have been committed to *master* and might be done in a Development branch or separate repository entirely. The goal for Experimental Platforms would be for them to eventually graduate to *Secondary* Platforms.
+*Supported* and *Experimental* Platforms remain active so long as there are Collaborators actively maintaining support for them.
 
-*Secondary* and *Experimental* Platforms remain active so long as there are Collaborators actively maintaining support for them.
+*Deprecated* Platforms consist of platforms for which: (a) there is no active development, (b) there are no Collaborators actively maintaining support, or (c) it is no longer feasible or practical to continue supporting.
 
-*Deprecated* Platforms consist of prior *Primary*, *Secondary* or *Experimental* Platforms that are no longer under active development and are no longer supported by active Collaborators. Only the TSC can decide to move a *Primary* platform to *Deprecated* status. Support for *Deprecated* Platforms may be removed entirely after the next *semver-major* version increase.
+It is possible for build and CI infrastructure updates to cause failures in existing *Supported* platforms. This can happen, for instance, when an updated dependency forces the build to use an updated toolchain that is not yet supported on a given platform. In such cases, Collaborators should be notified  in advance that such changes are expected so that appropriate fixes on affected platforms can be planned and landed. Effort should be made to ensure that the build and CI infrastructure continues to work on all currently *Supported* platforms. Prior to the next release, however, if it becomes clear that the build cannot be fixed on a particular platform, it can be moved to *Deprecated* status and landing the changes would require a *semver-major* version increase.
```

Comment:
> not yet supported on a given platform

More likely "no longer". It may be worthwhile noting, this doesn't mean we won't support it, but it may require external toolchain updates.